include/deal.II/grid/grid_tools.h

```diff
@@ -295,12 +295,19 @@ namespace GridTools
    * typically, each of the vertices at the boundary of the triangulation is
    * mapped to the corresponding points in the @p new_points map.
    *
-   * The way this function works is that it solves a Laplace equation for each
+   * The unknown displacement field $u_d(\mathbf x)$ in direction $d$ is obtained from
+   * the minimization problem
+   * \f[
+   *   \rm{min}\, \frac{1}{2}\int c(\mathbf x) \mathbf \nabla u_d(\mathbf x) \cdot \mathbf \nabla u_d(\mathbf x) \,\rm d v
```

Comment:
Can you break this line so that I get to see everything in the github patch review page? :-)